src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -0,0 +1,670 @@
+| numeric            | <span style={{ color: "blue" }}>Float</span>        |
+| **Date and Time**  |              |
+| date               | <span style={{ color: "firebrick" }}>AWSDate</span>      |
+| datetime           | <span style={{ color: "red" }}>AWSDateTime</span>   |
+| timestamp          | <span style={{ color: "red" }}>AWSDateTime</span>   |
+| time               | <span style={{ color: "deeppink" }}>AWSTime</span>       |
+| year               | <span style={{ color: "skyblue" }}>Int</span>          |
+| **Binary**         |              |
+| binary             | <span style={{ color: "orange" }}>String</span>        |
+| varbinary          | <span style={{ color: "orange" }}>String</span>        |
+| tinyblob           | <span style={{ color: "orange" }}>String</span>        |
+| blob               | <span style={{ color: "orange" }}>String</span>        |
+| mediumblob         | <span style={{ color: "orange" }}>String</span>        |
+| longblob           | <span style={{ color: "orange" }}>String</span>        |
+| **Others**         |              |
+| bool               | <span style={{ color: "green" }}>Boolean</span>       |
+| boolean            | <span style={{ color: "green" }}>Boolean</span>       |
+| bit                | <span style={{ color: "skyblue" }}>Int</span>          |
+| json               | <span style={{ color: "purple" }}>AWSJSON</span>       |
+| enum               | <span style={{ color: "violet" }}>ENUM</span>          |
+
+</Accordion>
+
+<BlockSwitcher>
+  <Block name="Amplify CLI">
+
+1. Make any adjustments to your SQL statements such as:
+
+```sql
+CREATE TABLE posts (
+  person_id       varchar(255)           NOT NULL PRIMARY KEY,
+  title           varchar(255)           NOT NULL,
+  content         varchar(255)           NOT NULL,
+  published       tinyint(1)  DEFAULT 0  NOT NULL
+  published_date  date                       NULL
+);
+```
+
+2. Generate an updated schema:
+
+```sh
+amplify api generate-schema
+```
+
+<Callout>
+  **NOTE:** If the connection to the database fails, the CLI will prompt you to
+  retry using a VPC endpoint.
+</Callout>
+
+3. Deploy your changes to the cloud:
+
+```sh
+amplify push -y
+```
+
+</Block>
+  <Block name="AWS CDK">
+  1. Make any adjustments to your SQL statements such as:
+
+```sql
+CREATE TABLE posts (
+  person_id       varchar(255)           NOT NULL PRIMARY KEY,
+  title           varchar(255)           NOT NULL,
+  content         varchar(255)           NOT NULL,
+  published       tinyint(1)  DEFAULT 0  NOT NULL
+  published_date  date                       NULL
+);
+```
+
+2. Generate an updated schema:
+
+```sh
+npx @aws-amplify/cli api generate-schema —sql-schema <sql-schema/my-schema.csv> —engineType mysql —out schema.sql.graphql
+```
+
+3. Deploy your changes to the cloud:
+
+```sh
+cdk deploy
+```
+
+</Block>
+</BlockSwitcher>
+
+## Make preserved changes to the GraphQL Schema
```

Comment:
Don't have a better suggestion but the header is slightly confusing

Reply:
Agree, I'll rethink the heading. I'm considering removing it altogether so that the nested sections - rename/map models, create custom queries/mutations, and create relationships are higher level in the right-side nav